### PR TITLE
Implement exercise 'isogram'

### PR DESCRIPTION
**Pull request by ingydotnet**

### CODE REVIEW

File: exercises/practice/isogram/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "authors": [],
```
Comment (m-dango) [inline exercises/practice/isogram/.meta/config.json]: You can add yourself as an author here if you'd like.

File: exercises/practice/isogram/test/test-1.ys [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- name: Same first and last characters
+  code: isogram?('angola')
+  want: false
+  uuid: 310ac53d-8932-47bc-bbb4-b2b94f25a83e
+
+- name: Word with duplicated character and with two hyphens
+  code: 
+  want: 
+  uuid: 0d0b8644-0a1e-4a31-a432-2b3ee270d847
+  SKIP: true
```
Comment (m-dango) [inline exercises/practice/isogram/test/test-1.ys]: Is this to be changed?

Comment (ingydotnet) [reply exercises/practice/isogram/test/test-1.ys]: > Is this to be changed?

Yep. Had to step away.

Comment (ingydotnet) [reply exercises/practice/isogram/test/test-1.ys]: Fixed in f37f8df

File: exercises/practice/isogram/.meta/config.json (hunk 1 above)
Comment (ingydotnet) [reply exercises/practice/isogram/.meta/config.json]: fixed in f37f8df

**Pull request merged**